File: Tests/experimental_spectra/test_R_Python_data_with_json.py
import json
from pprint import pprint

path='/Users/matteo/Documents/MassTodon/MassTodonPy/Tests/test_json_of_a_bitch.json'
with open(path) as data_file:
    data = json.load(data_file)

data['c'][0]
# Succcesss!!!

File: Tests/experimental_spectra/test_R_Python_data_with_json.R
suppressPackageStartupMessages(library("jsonlite"))

DATA = list(a='dupa','b'=10,c=rnorm(10))
DATA

toJSON(DATA, digits=10)
write_json(DATA, path='/Users/matteo/Documents/MassTodon/MassTodonPy/Tests/test_json_of_a_bitch.json', digits=10)
